WooCommerce/Classes/ViewRelated/Products/SKU Scanner/BarcodeScannerViewController.swift

```diff
@@ -0,0 +1,247 @@
+import AVFoundation
+import UIKit
+import Vision
+
+/// Starts live stream video for scanning barcodes.
+/// This view controller is meant to be embedded as a child view controller for navigation customization.
+final class BarcodeScannerViewController: UIViewController {
+    /// UI properties that could differ in each use case.
+    struct ViewProperties {
+        let instructionText: String
```

Comment:
Not at the moment, maybe in the future. do you think it's better to just pass this one property to the view controller instead of a struct now that there's only one?

Reply:
Not a big deal, but if it's only one property I would just pass one property. Up to you.

Reply:
makes sense! updated in https://github.com/woocommerce/woocommerce-ios/pull/3097/commits/a08089b4de36dba1ac05347d1ef67f9374322682